<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderSystemClass</class>
 <widget class="QMainWindow" name="OrderSystemClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1001</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OrderSystem</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="user_interface">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="2" column="2">
         <widget class="QPushButton" name="save_allinfo_button_1">
          <property name="text">
           <string>Сохранить информацию</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <layout class="QFormLayout" name="formLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Имя пользователя:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="userName_label">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <widget class="QToolButton" name="exit_to_register_menu">
          <property name="text">
           <string>Выйти из аккаунта</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="3">
         <widget class="QTabWidget" name="user_tab_widget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>Товары</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_9">
            <item row="0" column="0">
             <widget class="UserProductTable" name="user_product_table">
              <attribute name="horizontalHeaderDefaultSectionSize">
               <number>140</number>
              </attribute>
              <column>
               <property name="text">
                <string>Тип</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Наименование</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Цена($)</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Характеристики</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>ID</string>
               </property>
              </column>
             </widget>
            </item>
            <item row="1" column="0">
             <layout class="QGridLayout" name="gridLayout_6">
              <item row="0" column="0">
               <widget class="QToolButton" name="filter_button_user">
                <property name="text">
                 <string>Фильтры</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="backet_tab">
           <attribute name="title">
            <string>Корзина</string>
           </attribute>
           <widget class="QLabel" name="orderSum">
            <property name="geometry">
             <rect>
              <x>120</x>
              <y>342</y>
              <width>191</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>0</string>
            </property>
           </widget>
           <widget class="QPushButton" name="form_order">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>366</y>
              <width>98</width>
              <height>24</height>
             </rect>
            </property>
            <property name="text">
             <string>Оформить заказ</string>
            </property>
           </widget>
           <widget class="QLabel" name="label">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>340</y>
              <width>78</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>Сумма заказа:</string>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QLabel" name="create_order_error_label">
            <property name="geometry">
             <rect>
              <x>120</x>
              <y>370</y>
              <width>351</width>
              <height>16</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>9</x>
              <y>9</y>
              <width>921</width>
              <height>331</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="BacketTable" name="backet_table">
               <attribute name="horizontalHeaderDefaultSectionSize">
                <number>120</number>
               </attribute>
               <column>
                <property name="text">
                 <string>Тип</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Наименование</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Количество</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Цена($)</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>ID</string>
                </property>
               </column>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
          <widget class="QWidget" name="user_orders_tab">
           <attribute name="title">
            <string>Заказы</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <widget class="OrderTable" name="user_order_table">
              <attribute name="horizontalHeaderDefaultSectionSize">
               <number>170</number>
              </attribute>
              <attribute name="verticalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <column>
               <property name="text">
                <string>Состав</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Сумма($)</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Статус заказа</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>ID</string>
               </property>
              </column>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item row="2" column="1">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Register">
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>780</y>
          <width>829</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QWidget" name="formLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>120</y>
          <width>501</width>
          <height>166</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Зарегистрируйтесь:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Логин:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="singIn_login"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Пароль:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="singIn_password">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Повторите пароль:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="signIn_checkPassword">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QPushButton" name="signin_register_button">
           <property name="text">
            <string>Зарегистрироваться</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLabel" name="signIn_errorLabel">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QPushButton" name="signIn_back">
           <property name="text">
            <string>Вернуться назад</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="SingIn">
       <widget class="QWidget" name="formLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>250</y>
          <width>221</width>
          <height>31</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Нет аккаунта ?</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="reg_button">
           <property name="text">
            <string>Зарегистрироваться</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="formLayoutWidget">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>120</y>
          <width>501</width>
          <height>126</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Войдите в систему:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Логин:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="loginText">
           <property name="echoMode">
            <enum>QLineEdit::Normal</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Пароль:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="passwordText">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QPushButton" name="signin_button">
           <property name="text">
            <string>Войти</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="errorLogin_label">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>300</y>
          <width>271</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="enter_error_label">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>290</y>
          <width>571</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="admininterface">
       <widget class="QTabWidget" name="admin_tab_widget">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>9</y>
          <width>961</width>
          <height>461</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_4">
         <attribute name="title">
          <string>Товары</string>
         </attribute>
         <widget class="QToolButton" name="add_product_button">
          <property name="geometry">
           <rect>
            <x>9</x>
            <y>9</y>
            <width>99</width>
            <height>22</height>
           </rect>
          </property>
          <property name="text">
           <string>Добавить товар</string>
          </property>
         </widget>
         <widget class="QWidget" name="formLayoutWidget_5">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>400</y>
            <width>160</width>
            <height>31</height>
           </rect>
          </property>
          <layout class="QFormLayout" name="formLayout_5">
           <item row="0" column="0">
            <widget class="QToolButton" name="filter_button">
             <property name="text">
              <string>Фильтры</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="ProductTable" name="admin_product_table">
          <property name="geometry">
           <rect>
            <x>9</x>
            <y>37</y>
            <width>931</width>
            <height>361</height>
           </rect>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>140</number>
          </attribute>
          <column>
           <property name="text">
            <string>Тип</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Наименование</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Цена ($)</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Характеристики</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>ID</string>
           </property>
          </column>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_6">
         <attribute name="title">
          <string>Заказы</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="1" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="AdminOrderTable" name="admin_order_table">
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>170</number>
            </attribute>
            <column>
             <property name="text">
              <string>Состав</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Имя пользователя</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Стоимость ($)</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Статус заказа</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>ID</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Бухгалтерия</string>
         </attribute>
         <widget class="BankTable" name="bank_table">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>31</y>
            <width>921</width>
            <height>371</height>
           </rect>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>170</number>
          </attribute>
          <column>
           <property name="text">
            <string>Отправитель</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Получатель</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Сумма ($)</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Баланс после операции ($)</string>
           </property>
          </column>
         </widget>
         <widget class="QLabel" name="label_13">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>10</y>
            <width>101</width>
            <height>16</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Баланс:</string>
          </property>
         </widget>
         <widget class="QLabel" name="balance_label">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>10</y>
            <width>101</width>
            <height>16</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </widget>
       <widget class="QPushButton" name="exit_to_register_menu_admin">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>470</y>
          <width>131</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Выйти из аккаунта</string>
        </property>
       </widget>
       <widget class="QPushButton" name="save_allinfo_button">
        <property name="geometry">
         <rect>
          <x>800</x>
          <y>470</y>
          <width>151</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Сохранить информацию</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ProductTable</class>
   <extends>QTableWidget</extends>
   <header>ProductTable.h</header>
  </customwidget>
  <customwidget>
   <class>UserProductTable</class>
   <extends>QTableWidget</extends>
   <header>UserProductTable.h</header>
  </customwidget>
  <customwidget>
   <class>BacketTable</class>
   <extends>QTableWidget</extends>
   <header>BacketTable.h</header>
  </customwidget>
  <customwidget>
   <class>OrderTable</class>
   <extends>QTableWidget</extends>
   <header>OrderTable.h</header>
  </customwidget>
  <customwidget>
   <class>AdminOrderTable</class>
   <extends>QTableWidget</extends>
   <header>AdminOrderTable.h</header>
  </customwidget>
  <customwidget>
   <class>BankTable</class>
   <extends>QTableWidget</extends>
   <header>BankTable.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="OrderSystem.qrc"/>
 </resources>
 <connections/>
</ui>
